test/erc721.js

```diff
@@ -294,21 +276,17 @@ describe('ERC721', () => {
 	
 				it('non-owner of token shouldn`t be able to call approve', async () => {
 					//Arrange
-					// const deployContractPromise = deployedContract.call('mint', { args: `(${firstTokenId}, ${config.pubKeyHex})`, options: { ttl: config.ttl, gas: config.gas, nonce: nonces.first++ } })
+					// const deployContractPromise = deployedContract.call('mint', { args: `(${firstTokenId}, ${config.pubKeyHex})`, options: { ttl: config.ttl, gas: config.gas } })
```

Comment:
do we need this test? the whole code here is commented?

Reply:
No